### ISSUE TITLE
SPRACINGF3 No configuration received within 10 seconds, communication failed

### ISSUE THREAD

**Issue by nikola-v** [iNavFlight/inav#4520]
## Current Behavior
Cannot connect to Spracing F3 Deluxe.
No configuration received within 10 seconds, communication failed

## Steps to Reproduce
1. Put SPRACINGF3 to boot mode
2. Load flash SPRACINGF3 2.1.0.
3. Reboot SPRACINGF3  (boot pin is not connected)
4.  Open INAV configurator 2.1.4 and open connection

## Expected behavior
Configurator should open configuration for SPRACINGF3 but it just says:

2019-03-18 @ 19:56:24 -- Serial port successfully opened with ID: 74
2019-03-18 @ 19:56:34 -- No configuration received within 10 seconds, communication failed

## Suggested solution(s)
None

- FC Board name and vendor: SPRACINGF3 Deluxe
- INAV version string: Configurator 2.1.4

**Comment by shellixyz** [comment]
What OS are you running the configurator on ?

**Comment by nikola-v** [comment]
Windows 10 Pro
Chrome Version 72.0.3626.121 (Official Build) (64-bit)

**Comment by shellixyz** [comment]
Does it work with an older version of the firmware and configurator ? For example FW 1.9.1 and configurator 1.9.3: https://github.com/iNavFlight/inav-configurator/releases/download/1.9.3/INAV-Configurator_win32_1.9.3.zip

**Comment by shellixyz** [comment]
Also please use the standalone version of the configurator. Chrome version are not supported anymore.

**Comment by nikola-v** [comment]
Standalone configurator doesn't work, it opens up with black screen and stays that.
I have tried to run it as administrator but still it is black.
I have tried with latest cleanflight / cleanflight configurator and it works without issues.
INAV 1.9.3 as you proposed in standalone version also opens up black (just black window), also tried to run it with administrator perm but nothing changed.

![image](https://user-images.githubusercontent.com/11407227/54671533-634e5900-4af6-11e9-9d61-67027cf859be.png)
